sections/advanced/reaching-ui-state.md

```diff
@@ -0,0 +1,222 @@
+# Reaching UI state for tests without using the UI
+
+<br/><br/>
+
+### One Paragraph Explainer
+
+It adds value to cover a UI scenario *once*, and it provides little value to duplicate any parts of it in other tests; tests which may require a relevant state of the system. Suppose in a new test you require a state, and that state -partially or in full- duplicates testing from a UI test. Scenarios like this are excellent for utilizing certain techniques:
+
+* Direct navigation
+* Network stub record & play
+* Application actions
+* Seeding the database
+
+> Disclaimer: applications of the whole package of these techniques are only possible in Cypress (as far as we know), consequently the code samples below are are in Cypress context.
+
+<br/><br/>
+
+### Direct navigation
+
+This is the easiest technique and is applicable in any framework. Suppose the intent of the test is concerned with a certain page in your application. Instead of click navigating, directly visit the URL. Upon landing you can either wait for a UI element (any test framework) or network calls (some test frameworks), or both.
+
+```javascript
+// Test A covers click-navigation to a certain page.
+// This is Test B, and navigating to that page is the prerequisite step.
+
+// assuming baseUrl is set in cypress.json or config file
+// directly navigate to the page.
+cy.visit('/endpoint');
+
+// to ensure stability, wait for network (preferred), ui elements, or both
+
+// note: you can optionally add a url check
+cy.url().should('contain', 'endpoint');
```

Comment:
Why this assertion? The reader could be confused because you don't explain to him/her the reason

Reply:
added clarifying comments